packages/perseus-editor/src/__docs__/article-editor.stories.tsx

```diff
@@ -54,3 +55,32 @@ export const Demo = (): React.ReactElement => {
         </View>
     );
 };
+
+export const WithEditingDisabled = (): React.ReactElement => {
+    const articleEditorRef = useRef();
+    const disabledApiOptions = {
+        ...ApiOptions.defaults,
+        editingDisabled: true,
+    };
+
+    function serialize() {
+        // eslint-disable-next-line no-console
+        console.log((articleEditorRef.current as any).serialize());
+    }
```

Comment:
Do we need this? I remember @ivyolamit removed the `serialize` button in the updated article editor story?

Reply:
Oooo great! I think this snuck along due to some edits -- I'll remove it!